server/models/Collective.js

```diff
@@ -3045,9 +3045,8 @@ function defineModel() {
     if (!this.isHostAccount || !this.isActive || this.type !== types.ORGANIZATION) {
       return null;
     }
-
-    from = from ? moment(from) : moment().utc().startOf('month');
-    to = to ? moment(to) : moment(from).utc().endOf('month');
+    from = from ? moment(from) : null;
+    to = to ? moment(to) : null;
```

Comment:
You're not updating the inline documentation above?

Reply:
Changing the behavior is gonna obviously break the previous Host Metrics panel (related https://github.com/opencollective/opencollective/issues/4989). What's your strategy around this?

Reply:
> You're not updating the inline documentation above?

Ops, missed that. I've updated it now. 🤔

> Changing the behavior is gonna obviously break the previous Host Metrics panel (related [opencollective/opencollective#4989](https://github.com/opencollective/opencollective/issues/4989)). What's your strategy around this?

Thanks for bringing this up. Something I've overlooked; I'll do a PR today to tackle https://github.com/opencollective/opencollective/issues/4989 so we can merge these two PRs together. Hope that sounds good? 🤔

Reply:
> What's your strategy around this?

Reply:
My thinking was that we show the message like this https://github.com/opencollective/opencollective-frontend/pull/7246 and only show the metrics in one place (which is the new dashboard). Or is there a specific need that we need to show the host metrics in the old host metrics page as well until we completely remove it? 🤔

Reply:
@znarf : Now that https://github.com/opencollective/opencollective-frontend/pull/7246 is merged I think we are good to go with this one?

Reply:
Ok